### PR TITLE
Migrate to Github Actions

### PR DESCRIPTION
**Pull request by michaljurecko**
Slack: https://keboola.slack.com/archives/CQ1ASK06M/p1644415921850459?thread_ts=1644389488.688299&cid=CQ1ASK06M

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+name: GitHub Actions
+on: [ push ]
+concurrency: ci-${{ github.ref }} # to avoid tag collisions in the ECR
```
Comment (webrouse) [inline .github/workflows/push.yml:3]: Premigrovane na GHA.

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+name: GitHub Actions
+on: [ push ]
```
Comment (michaljurecko) [inline .github/workflows/push.yml:2]: Premigrovane na GHA.

File: tests/datadir/search-sharing-link-not-found/expected-stderr [hunk 3]
```diff
@@ -1 +1 @@
-The sharing link "https://example.com/a/b/c/not/fo..." is invalid.
+The sharing link "https://example.com/a/b/c/not/fo..." no exists, or you do not have permission to access it.
```
Comment (michaljurecko) [inline tests/datadir/search-sharing-link-not-found/expected-stderr:1]: Fixnuty test, zmenila sa odpoved API.

**Review by ondrajodas — APPROVED** [review]
👍

**Pull request merged**